### PR TITLE
Change style guide to prefer single-quoted strings

### PR DESCRIPTION
**Pull request by ryanjulian**
* Single quotes seem to be more popular in general among the Python community
e.g. https://www.kennethreitz.org/essays/if-i-could-amend-pep-8)
* There is automated tooling for single quotes, but not double quotes

### CODE REVIEW

**Comment by ryanjulian** [comment]
Alternative rule (which I prefer, but I think is hard to enforce):
Use single-quotes for strings which are meant for machine consumption (arguments, etc.) and double-quotes for strings meant for interpolation and human consumption (e.g. error messages).

**Comment by ryanjulian** [comment]
https://github.com/rlworkgroup/akro/pull/12

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The Python code in garage conforms to the [PEP8](https://www.python.org/dev/peps
 ### garage-specific Python style
 These are garage-specific rules which are not part of the aforementioned style guides.
 * Python package imports should be sorted alphabetically within their PEP8 groupings. The sorting is alphabetical from left to right, ignoring case and Python keywords (i.e. `import`, `from`, `as`). Notable exceptions apply in `__init__.py` files, where sometimes this rule will trigger a circular import.
-* We prefer double-quoted strings (`"foo"`) over single-quoted strings (`'foo'`), unless there is a compelling escape or formatting reason for using single quotes
+* We prefer single-quoted strings (`'foo'`) over single-quoted strings (`"foo"`), unless there is a compelling escape or formatting reason for using single quotes (e.g. a single quote appears inside the string).
```
Comment (CatherineSue) [inline CONTRIBUTING.md]:
``` suggestion
We prefer single-quoted strings (`'foo'`) over double-quoted strings (`"foo"`), unless there is a compelling escape or formatting reason for using double quotes (e.g. a single quote appears inside the string).
```

Comment (krzentner) [reply CONTRIBUTING.md]: Good catch.

**Pull request merged**